packages/ripple-nuxt-tide/lib/pages/Tide.vue

```diff
@@ -65,20 +65,15 @@
 </template>
 
 <script>
-// Tide
-import RplAccordion from '@dpc-sdp/ripple-accordion'
```

Comment:
RplAccordion has been imported by dynamic component, this is redundant and increased global CSS/JS size.